Kickstart.sh doesn't work for CentOS 8 & RHEL 8

<!---
When creating a bug report please:
- Verify first that your issue is not already reported on GitHub
- Test if the latest release and master branch are affected too.
- Provide a clear and concise description of what the bug is in "Bug report 
  summary" section.
- Try to provide as much information about your environment (OS distribution,
  running in container, etc.) as possible to allow us reproduce this bug faster.
- Write which component is affected. We group our components the same way our
  code is structured so basically:
  component name = dir in top level directory of repository
- Describe how you found this bug and how we can reproduce it. Preferable with
  a minimal test-case scenario. You can paste gist.github.com links for larger
  files
- Provide a clear and concise description of what you expected to happen.
-->

##### Bug report summary
Running the `kickstart.sh` on CentOS 8 fails on 'Installation of required packages'.

```
/etc/os-release information:
NAME            : CentOS Linux
VERSION         : 8 (Core)
ID              : centos
ID_LIKE         : rhel fedora
VERSION_ID      : 8

We detected these:
Distribution    : centos
Version         : 8
Codename        : 8 (Core)
Package Manager : install_yum
Packages Tree   : centos
Detection Method: /etc/os-release
Default Python v: 2

This system is receiving updates from Red Hat Satellite or Spacewalk server.
Last metadata expiration check: 2:42:06 ago on Fri Sep 27 14:58:16 2019.
No match for argument: Judy-devel
No match for argument: MySQL-python
No match for argument: autoconf-archive
No match for argument: autogen
No match for argument: libmnl-devel
No match for argument: libuv-devel
No match for argument: python
There are following alternatives for "python": python2, python36
No match for argument: python-psycopg2



We are very sorry!

Installation of required packages failed.

What to do now:

1. Make sure your system is updated.
Most of the times, updating your system will resolve the issue.


2. If the error message is about a specific package, try removing
that package from the command and run it again.
Depending on the broken package, you may be able to continue.


3. Let us know. We may be able to help.
Open a github issue with the above log, at:


https://github.com/firehol/netdata/issues                                                                                             
```
##### OS / Environment
CentOS 8
##### Netdata version (ouput of `netdata -V`)
1.10.0 (I am limited to this version due to company policies. Version should be irrelevant to this issue) 
##### Component Name
Installation scripts
##### Steps To Reproduce
`bash kickstart.sh all --non-interactive`
##### Expected behavior
Installation should succeed.

For what it's worth, I get the same error when trying the auto-installer for the latest version on CENTOS 6:
```
configure: error: libuv required but not found. Try installing 'libuv1-dev' or 'libuv-devel'.
 FAILED


  ^
  |.-.   .-.   .-.   .-.   .-.   .  netdata                          .-.   .-
  |   '-'   '-'   '-'   '-'   '-'   sorry, it failed to build...  '-'   '-'
  +----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+--->


^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Sorry! netdata failed to build...

You may need to check these:

1. The package uuid-dev (or libuuid-devel) has to be installed.

   If your system cannot find libuuid, although it is installed
   run me with the option:  --libs-are-really-here

2. The package zlib1g-dev (or zlib-devel) has to be installed.

   If your system cannot find zlib, although it is installed
   run me with the option:  --libs-are-really-here

3. You need basic build tools installed, like:

   gcc make autoconf automake pkg-config

   Autoconf version 2.60 or higher is required.

If you still cannot get it to build, ask for help at github:

   https://github.com/netdata/netdata/issues
```

Installing with the precompiled x64 binaries worked fine

--- a/packaging/installer/methods/manual.md
+++ b/packaging/installer/methods/manual.md
@@ -117,7 +117,7 @@ Netdata DB engine can be enabled when these are installed (they are optional):
 
 ### CentOS / RHEL 8.x
 
-For CentOS / RHEL 8.x a lot of development packages have moved out into thwi
+For CentOS / RHEL 8.x a lot of development packages have moved out into their
 own separate repositories. Some other dependeicies are either missing completely
 or have to be sourced by 3rd-parties.
 

--- a/packaging/installer/methods/manual.md
+++ b/packaging/installer/methods/manual.md
@@ -31,6 +31,7 @@ should be installed on your system to build and run Netdata. It supports most ma
         [EPEL](http://www.tecmint.com/how-to-enable-epel-repository-for-rhel-centos-6-5/).
         In addition, RHEL/CentOS version 6 also need
         [OKay](https://okay.com.mx/blog-news/rpm-repositories-for-centos-6-and-7.html) for package libuv version 1.
+    -   CentOS 8 / RHEL 8 requires a bit of extra work. See the dedicated section below.
 
 -   **SuSe** Linux and its derivatives (including **openSuSe**)
 
@@ -113,6 +114,40 @@ Netdata DB engine can be enabled when these are installed (they are optional):
 | `openssl`| Cryptography and SSL/TLS toolkit|
 
 *Netdata will greatly benefit if you have the above packages installed, but it will still work without them.*
+
+### CentOS / RHEL 8.x
+
+For CentOS / RHEL 8.x a lot of development packages have moved out into thwi
+own separate repositories. Some other dependeicies are either missing completely
+or have to be sourced by 3rd-parties.
+
+CentOS 8.x:
+
+- Enable the PowerTools repo
+- Enable the EPEL repo
+- Enable the Extra repo from [extra.getpagespeed.com](https://extras.getpagespeed.com/release-el8-latest.rpm)
+
+And install the minimum required dependencies:
+
+```sh
+# Enable config-manager
+yum install -y 'dnf-command(config-manager)'
+
+# Enable PowerTools
+yum config-manager --set-enabled PowerTools
+
+# Enable EPEL
+yum install -y epel-release
+
+# Install Repo for libuv-devl (NEW)
+yum install -y https://extras.getpagespeed.com/release-el8-latest.rpm
+
+# Install Devel Packages
+yum install autoconf automake curl gcc git libmnl-devel libuuid-devel openssl-devel libuv-devel lz4-devel make nc pkgconfig python3 zlib-devel
+
+# Install Judy-Devel directly
+yum install -y http://mirror.centos.org/centos/8/PowerTools/x86_64/os/Packages/Judy-devel-1.0.5-18.module_el8.1.0+217+4d875839.x86_64.rpm
+```
 
 ---
 

